lib/license_scout/overrides.rb

```diff
@@ -671,6 +671,7 @@ def default_overrides
         ["debug", "MIT", ["https://raw.githubusercontent.com/visionmedia/debug/master/LICENSE"]],
         ["performance-now", "MIT", ["https://raw.githubusercontent.com/braveg1rl/performance-now/master/license.txt"]],
         ["cli-table", "MIT", ["README.md"]],
+        ["process", "MIT", ["https://raw.githubusercontent.com/defunctzombie/node-process/master/LICENSE"]],
```

Comment:
should i switch this to https://raw.githubusercontent.com/defunctzombie/node-process/v0.5.2/LICENSE ?